src/components/MoneyRequestConfirmationList.js

```diff
@@ -140,18 +113,8 @@ class MoneyRequestConfirmationList extends Component {
      * @returns {Array}
      */
     getParticipantsWithAmount(participants) {
-        const iouAmount = IOUUtils.calculateAmount(participants.length, this.props.iouAmount);
-        return OptionsListUtils.getIOUConfirmationOptionsFromParticipants(participants, CurrencyUtils.convertToDisplayString(iouAmount, this.props.iou.selectedCurrencyCode));
-    }
-
-    /**
-     * Returns the participants without amount
-     *
-     * @param {Array} participants
-     * @returns {Array}
-     */
-    getParticipantsWithoutAmount(participants) {
```

Comment:
Just trying to reduce unnecessary code. Previously, we initially map all participants with amount [here](https://github.com/Expensify/App/pull/17964/files#diff-dc0f934ad9e583b0f7ac049563a3dcacbfe314e9c16b5c6f8bba6e5a5225a86cL91-L94). But, if you realize, `getParticipantsWithAmount` just append a `descriptiveText` property to be shown in the section list and actually we already call `getParticipantsWithAmount` in `getSections` [here](https://github.com/Expensify/App/pull/17964/files#diff-dc0f934ad9e583b0f7ac049563a3dcacbfe314e9c16b5c6f8bba6e5a5225a86cL168). Because we initially map all participants with `descriptiveText`, `getParticipantsWithoutAmount` is being used to remove that if we don't want the `descriptiveText` like [here](https://github.com/Expensify/App/pull/17964/files#diff-dc0f934ad9e583b0f7ac049563a3dcacbfe314e9c16b5c6f8bba6e5a5225a86cL169).

Reply:
So we used to call `getParticipantsWithoutAmount` on objects that never had the `descriptiveText`? Maybe there were some cases where the passed props `participants` had such values.

So `getParticipantsWithAmount` was used not to just append, but to overwrite `descriptiveText` as well. Where `getParticipantsWithoutAmount` was used to omit it.

Reply:
> on objects that never had the descriptiveText

Yes, `participants` props don't have `descriptiveText`. Logged [here](https://github.com/Expensify/App/pull/17964/files#diff-879f243892ee617d3ea9bc2550496a07521c2b91d2bec3eb12b90f09aac6a05aR97-R101), [here](https://github.com/Expensify/App/pull/17964/files#diff-a251643c8dcdcae382baa17a80b39f5420f734b50700507979ce503624b9545eR144), and [here](https://github.com/Expensify/App/pull/17964/files#diff-a9d618fe51be58d446a3e60b2ab84d7a01dcce944ef738335bf83745b3b99297R181)

Reply:
In that case, please mark this as resolved.